src/components/CustomisedBulletpoint.js

```diff
@@ -4,15 +4,23 @@ import is from 'styled-is'
 
 export default styled.li`
   position: relative;
-  padding-bottom: ${remcalc(47)};
-  margin-bottom: ${remcalc(30)};
+  padding-top: ${remcalc(12)};
+  padding-bottom: ${remcalc(21)};
+  margin-bottom: ${remcalc(8)};
   line-height: ${remcalc(24)};
 
   ${is('symmetrical')`
+    padding-top: ${remcalc(0)};
     padding-bottom: ${remcalc(30)};
     margin-bottom: ${remcalc(18)};
   `};
 
+  ${is('big')`
```

Comment:
big seems like it will turn the `li` bigger, but it's really increasing the margin. Is `bottomSpaced` a better name? Not even sure..

Reply:
I will change to spaced